Fix signature required checkbox not updating prices for label requests.

tommyshellberg opened the pull request:
## Description
Ensure that the correct number of arguments are passed to the parent ShippingRates component when checking "require signature" option when requesting a shipping label.

### Related issue(s)

Closes #2185 

### Steps to reproduce & screenshots/GIFs

See the screencast in the original issue:

https://github.com/Automattic/woocommerce-services/issues/2185#issue-702784563

With this branch checked out you should see the prices updating when adding a signature:

![Screen Capture on 2020-11-18 at 12-30-49](https://user-images.githubusercontent.com/6723003/99545663-eb64c000-29b5-11eb-9da8-3d59d87e5ff2.gif)


### Checklist

<!-- All testable code should have tests. -->
- [x] unit tests

<!-- An entry in the changelog file is always required -->
- [x] `changelog.txt` entry added

bborman22 reviewed (commented):
I found that when I switched services from a signature required to a new service and switched back, the sidebar did not maintain the signature required cost. I pushed up a commit to address this. If this looks good to you let me know and I'll approve the PR! 

One other thought I had while looking at this was wondering if maybe the signature required checkboxes should be "cleared" when changing services? I didn't look into what would be required for that, but thought I'd ask your opinion on that approach as well?

tommyshellberg commented:
> I found that when I switched services from a signature required to a new service and switched back, the sidebar did not maintain the signature required cost. I pushed up a commit to address this. If this looks good to you let me know and I'll approve the PR!

Thanks for catching this! The changes work and make sense to me. 🚢 

> One other thought I had while looking at this was wondering if maybe the signature required checkboxes should be "cleared" when changing services? I didn't look into what would be required for that, but thought I'd ask your opinion on that approach as well?

I think from a UX perspective it would be nice to actually keep the values when switching. I can imagine a shop owner adding extras to the available shipping services and switching between them to compare prices; in that case, they'd want to keep the selection.  🤷

bborman22 reviewed (approved):
Everything worked as expected with no issues!
